Add psql guide (#506)
timescale/docs: timescaledb/how-to-guides/psql-basics.md

> # Psql Basics

Reviewer: ```suggestion
# Psql basic commands
```
Not sold on 'basic' but struggling to come up with an alternative.

Psql is the terminal-based front end to PostgreSQL, and is the primary tool used
to communicate with your TimescaleDB instances. Below is a refresher on some essential
psql commands that you may come across in our documentation, and will find useful
as you explore PostgreSQL and TimescaleDB.

For an in-depth breakdown of all commands, visit [psql's documentation](https://www.postgresql.org/docs/13/app-psql.html).

## Connections

> * Connect to a PostgreSQL/TimescaleDB server via psql (using parameter flags)

Reviewer: ```suggestion
* Connect to a PostgreSQL/TimescaleDB server via psql (using parameter flags):
```

```bash
psql -h host_name -p port -U username -W -d database_name
```

> * Connect to a PostgreSQL/TimescaleDB server via psql (using a service URL)

Reviewer: ```suggestion
* Connect to a PostgreSQL/TimescaleDB server via psql (using a service URL):
```

(and so on down 👇  )

```bash
psql "your_service_URL"
```

* Show your psql version
```bash
psql --version
```

## Functions

Edit a function

```bash
\ef function_name
```

Display all functions

```bash
\df
```

Display all functions with more details (including owner, source code, and description)
```bash
\df+
```

## Views

List views from current schema

```bash
\dv
```

List views from all schemas
```bash
\dv+
```

## Users

Display PostgreSQL database roles
```bash
\du
```

> Reset the password for a PostrgreSQL database user

Reviewer: ```suggestion
Reset the password for a PostgreSQL database user
```

```bash
\password username
```

> ## Indexes and Extensions

Reviewer: ```suggestion
## Indexes and extensions
```

List all indexes from all tables

```bash
\di
```

Show all extensions installed
```bash
\dx
```

> ## Import and Export CSV

Reviewer: ```suggestion
## Import and export CSV
```

> The following commands can be used to import csv files and export csv files from your PostgreSQL or TimescaleDB database.

Reviewer: ```suggestion
The commands in this section can be used to import and export CSV files from your PostgreSQL or TimescaleDB database.
```

<terminal>

<tab label="Import">

> ```

Reviewer: ```suggestion
```sql
```

??

\copy table_name FROM '/path/to/data.csv' DELIMITER ',' CSV HEADER
```

</tab>

<tab label="Export">

```
\copy (SELECT * FROM table_name) TO 'output_file.csv' WITH CSV HEADER

```

</tab>

</terminal>

> ## Backup and Restore

Reviewer: ```suggestion
## Backup and restore
```

> Note: the following commands utilize the pg_dump tool and pg_dumpall tools.

Reviewer: ```suggestion
<highlight type="note>
The commands in this section use the `pg_dump` and `pg_dumpall` tools.
</highlight>
```

Create a backup for a specific database in plan-text SQL script file
   (with default backup options)

> ```

Reviewer: ```suggestion
```sql
```

And so on down 👇

# Psql Basics

Psql is the terminal-based front end to PostgreSQL, and is the primary tool used
to communicate with your TimescaleDB instances. Below is a refresher on some essential
psql commands that you may come across in our documentation, and will find useful
as you explore PostgreSQL and TimescaleDB.

For an in-depth breakdown of all commands, visit [psql's documentation](https://www.postgresql.org/docs/13/app-psql.html).

## Connections

> 1. Connect to a PostgreSQL/TimescaleDB server via psql (using parameter flags)

Author: Great point!
